core/trino-parser/src/main/java/io/trino/sql/SqlFormatter.java

```diff
@@ -1366,6 +1367,26 @@ protected Void visitDropColumn(DropColumn node, Integer context)
             return null;
         }
 
+        @Override
+        protected Void visitTableExecute(TableExecute node, Integer indent)
+        {
+            builder.append("ALTER TABLE ");
+            builder.append(formatName(node.getTable().getName()));
+            builder.append(" EXECUTE ");
+            builder.append(node.getProcedureName());
```

Comment:
should go thru `io.trino.sql.ExpressionFormatter#formatIdentifier`
(unless we decide to constraint set of legal procedure names)

Reply:
I would rather constraint to just letters and assume case insensitivity. Not sure how to do that best though.

Reply:
Actually it was handling it correct I think as `Identifier.toString` calls out to `formatExpression`. But I will call it explicitly so it is more obvious when read.